ansible/roles/zookeeper/templates/zoo.cfg

```diff
@@ -15,6 +15,10 @@ clientPort={{ zookeeper_client_port }}
 # the maximum number of client connections.
 # increase this if you need to handle more clients
 maxClientCnxns=100
+# support zk monitoring
+{% if zookeeper_version.startswith('3.5') %}
+4lw.commands.whitelist=mntr
+{% endif %}
```

Comment:
I assume this config was added new in 3.5, but it is probably ignored in earlier versions. If so, there's probably not a need for the conditional, and I would instead just put update the comment:

```ini
# support zk monitoring (for zk >= 3.5)
```